app/src/main/java/chat/rocket/android/helper/BackStackHelper.java

```diff
@@ -0,0 +1,9 @@
+package chat.rocket.android.helper;
+
+/**
+ * Created by whocares on 28/12/17.
+ */
+
```

Comment:
@aniketsingh03 Please always remove the `author` doc. It's not helpful at all and we can keep track of the changes everyone does by using git blame or anything like that.